src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableQuery.cs

```diff
@@ -110,9 +110,9 @@ private IEnumerator<T> GetEnumerator()
             return _enumerable.GetEnumerator();
         }
 
-        public override string ToString()
+        public override string? ToString()
```

Comment:
__expression.ToString() wouldn't return null (I am still working on Linq.Expression)

Reply:
please disregard above comment, seems you make it nullable because of _enumerable.ToString() 

_enumerable.ToString()  is calling object.ToString() which is calling Type.ToString() which will not be nullable. (We made object.toString() nullable because it is abstract and there can be overrides returning null )